redis/acknowledge.lua

```diff
@@ -1,8 +1,7 @@
 local zset_key = KEYS[1]
 local processed_key = KEYS[2]
 
-local worker_id = ARGV[1]
-local test = ARGV[2]
+local test = ARGV[1]
```

Comment:
Makes sense 👍 So how can I look at some code and say "this part would be better if it were written in lua"?